lib/ash_authentication_phoenix/overrides/daisy_ui.ex

```diff
@@ -0,0 +1,196 @@
+  end
+
+  override ResetLive do
+    set :root_class, "grid h-screen place-items-center bg-base-100"
+  end
+
+  override Components.Reset do
+    set :root_class, """
+    flex-1 flex flex-col justify-center py-12 px-4 sm:px-6 lg:flex-none
+    lg:px-20 xl:px-24
+    """
+
+    set :strategy_class, "mx-auto w-full max-w-sm lg:w-96"
+  end
+
+  override Components.Reset.Form do
+    set :root_class, nil
+    set :label_class, "mt-2 mb-4 text-2xl tracking-tight font-bold text-base-content"
+    set :form_class, nil
+    set :spacer_class, "py-1"
+    set :button_text, "Change password"
+    set :disable_button_text, "Changing password ..."
+  end
+
+  override Components.SignIn do
+    set :root_class, """
+    flex-1 flex flex-col justify-center py-12 px-4 sm:px-6 lg:flex-none
+    lg:px-20 xl:px-24
+    """
+
+    set :strategy_class, "mx-auto w-full max-w-sm lg:w-96"
+
+    set :authentication_error_container_class, "text-base-content text-center"
+    set :authentication_error_text_class, ""
+    set :strategy_display_order, :forms_first
+  end
+
+  override Components.Banner do
+    set :root_class, "w-full flex justify-center py-2"
+    set :href_class, nil
+    set :href_url, "/"
+    set :image_class, "block dark:hidden"
+    set :dark_image_class, "hidden dark:block"
+    set :image_url, "https://ash-hq.org/images/ash-framework-light.png"
+    set :dark_image_url, "https://ash-hq.org/images/ash-framework-dark.png"
+    set :text_class, nil
+    set :text, nil
+  end
+
+  override Components.HorizontalRule do
+    set :root_class, "divider my-2"
+    set :hr_outer_class, "hidden"
+    set :hr_inner_class, "hidden"
+    set :text_outer_class, "contents"
+    set :text_inner_class, "contents"
+    set :text, "or"
+  end
+
+  override MagicSignInLive do
+    set :root_class, "grid h-screen place-items-center bg-base-100"
+  end
+
+  override Components.MagicLink do
+    set :root_class, "mt-4 mb-4"
+    set :label_class, "mt-2 mb-4 text-2xl tracking-tight font-bold text-base-content"
+    set :form_class, nil
+
+    set :request_flash_text,
+        "If this user exists in our database, you will be contacted with a sign-in link shortly."
+
+    set :disable_button_text, "Requesting ..."
+  end
+
+  override Components.MagicLink.Input do
+    set :submit_class, "btn btn-primary btn-block mt-4 mb-4"
+  end
+
+  override Components.Password do
+    set :root_class, "mt-4 mb-4"
+    set :interstitial_class, "flex flex-row justify-between content-between text-sm font-medium"
+
+    set :toggler_class,
+        "flex-none text-primary hover:text-primary-focus px-2 first:pl-0 last:pr-0"
+
+    set :sign_in_toggle_text, "Already have an account?"
+    set :register_toggle_text, "Need an account?"
+    set :reset_toggle_text, "Forgot your password?"
+    set :show_first, :sign_in
+    set :hide_class, "hidden"
+    set :register_form_module, AshAuthentication.Phoenix.Components.Password.RegisterForm
```

Comment:
There are a couple of keys missing here - https://github.com/team-alembic/ash_authentication_phoenix/blob/main/lib/ash_authentication_phoenix/overrides/default.ex#L137-L138 - do they need to be included?

Reply:
Yeah, looks like those were added after I copied. :)
I'll see if I can put together a test to verify there's no missing keys.